scripts/travis_build.sh

```diff
@@ -0,0 +1,4 @@
+#!/usr/bin/env bash
+set -e
+bundle exec img_checker
```

Comment:
Since it's a single line, add it directly to Travis.yml

Reply:
How will I do that?

Reply:
I researched but found nothing in the docs